src/lib/style-properties.html

```diff
@@ -285,6 +285,13 @@
         return {properties: props, key: o};
       },
 
+      _rootSelector: /(?:^:root)|(?::host\s*>\s*\*)/,
```

Comment:
Shouldn't we never have `:root` at this time? also, this is a full (potentially complex) selector so starting form the beginning only is potentially bad and unnecessary, right?